package.json

```diff
@@ -70,7 +69,7 @@
     "ts-node": "^10.2.0",
     "tsc-watch": "^4.2.9",
     "tslib": "^2.3.1",
-    "typescript": "^5.4.5",
+    "typescript": "~4.4.2",
```

Comment:
I think we can keep this at the 5.4.5 version unless there's a good reason not to - I need to bump the ts version anyways for https://github.com/magiclabs/magic-js/pull/742 to minimum `4.5.2`

Reply:
hmm, then I will set to 4.5.2 [c0b30c3](https://github.com/magiclabs/magic-js/pull/758/commits/c0b30c3bbcace66b858132e0fa57bd9064de05f9)
It seems that typescript-eslint is throwing a lot of errors when upgrading to version 5, and it would require a lot of changes to fix it.